lib/std/smartptrs.nim

```diff
@@ -0,0 +1,212 @@
+#
+#
+#            Nim's Runtime Library
+#        (c) Copyright 2021 Nim contributors
+#
+#    See the file "copying.txt", included in this
+#    distribution, for details about the copyright.
+
```

Comment:
wrapping this in `when defined(nimExperimentalSmartptrs):` is IMO justified

Reply:
ok

Reply:
fixed